userdocs/profiles.dev/docs/catalog-docs/add-profiles.md

```diff
@@ -39,15 +39,18 @@ spec:
   - url: https://github.com/weaveworks/profiles-examples
 ```
 
-After applying the manifest and waiting a moment, we can use `pctl search` to see the
+After applying the manifest and waiting a moment, we can use `pctl get` to see the
 catalogued profiles:
 
 ```bash
 $ kubectl apply -f dynamic-catalog-source.yaml
 # allow a few moments. the more profiles/tags, the more time the catalog manager
 # will need to discover them all
 
-$ pctl search --all
+$ pctl get 
+INSTALLED PACKAGES
```

Comment:
should this be `pctl get --catalog` since before it was just a search?

Reply:
`pctl search --all` is now `pctl get`

Reply:
but search --all only showed catalog entries right?

Reply:
oh i get it now, the above line is talking about only catalog profiles. i have fixed it. :D